src/css/application.css

```diff
@@ -517,6 +517,22 @@ body {
   opacity: 0.5;
 }
 
+.preview__reset-button {
```

Comment:
It seems like this shares enough with `.preview__pop-out-button` to justify a shared component (e.g. `preview__button` with modifiers `preview__button_pop-out` and `preview__button_reset`)